### DISCUSSION TITLE
How to input HLM CLint in PK-sim

### DISCUSSION THREAD

**Discussion by jyna6** [Open-Systems-Pharmacology/Forum#1762] [Questions & Problems]
I want to add the HLM CLint (unit: uL/min/pmol) in my PK-sim.
In PK-sim when I click 'Total hepatic Clearance -> In vitro liver microsomes' I can't find any blank to put the HLM CLint value.
Can anyone help me for this? Or is there any other way to put this information in PKsim?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/152915373/8e2f8e55-12bd-4ae0-af28-2150b0574a17)

**Comment by tobiasK2001** [comment]
Hi, 
if have a 1/2 from microsomal incubations you can use the input you show above. 
If you have a already physiologically scaled Clint you have to go via the liver plasma CL like this: i.e in put as specific Clearance via dividing by bodyweight and liver volume ( the liver mass assuming density is 1). Here is an example calculation for a literature value of e.g. midazolam:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/96cd415d-9f9d-46bc-b412-0266494c7b2d)

Hope this might help. 
Best, Tobias

**Reply by ikun-cai98** [reply]
Hi,
Can you provide a reference for the specific clearance formula?

**Comment by Proen1** [comment]
Hi, 
The units of your clearance  uL/min/pmol seem to be form a recombinant enzyme assay since it has pmol. 
If it is recombinant enzyme you need the option :
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/99483346/c718e57b-262a-463d-b1e2-904fa698dc80)

 Usually microsomal clearance is in uL/min/mg protein.  Does your source report somehow what is the pmol referring to?
